app/templates/admin/settings/images.hbs

```diff
@@ -1 +1 @@
-{{forms/admin/settings/images-form images=model save='saveImages' isLoading=isLoading}}
+{{forms/admin/settings/images-form eventImage=model.event speakerImage=model.speaker save='saveImages' isLoading=isLoading}}
```

Comment:
instead of passing them separately you can pass the model as `model` and later within the component you can refer to them as `model.eventImageSizes` and `model.speakerImageSizes`